src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -68,6 +68,22 @@ public async Task<IEnumerable<ReviewListItemModel>> GetReviewsAsync(string langu
             return reviews;
         }
 
+        public async Task<string> GetLanguageReview(string reviewId)
```

Comment:
Why do you need this API? You can get review model using review Id and it has the language property?